### PR TITLE
updated schemas

### PR DESCRIPTION
**Pull request by sudhirverma**

### CODE REVIEW

**Review by lstocchi — APPROVED** [review]
LGTM

File: src/yaml-support/tkn-yaml-scheme-generator.ts [hunk 1]
```diff
@@ -90,12 +89,12 @@ function injectResourceName(templateObj: any, resNames: string[]): {} {
...
-    templateObj.definitions.PipelineTaskCondition.properties.conditionRef.enum = conditions;
-  }
-  return templateObj;
-}
+// function injectConditionRefs(templateObj: any, conditions: string[]): {} {
+//   if (conditions && conditions.length > 0) {
+//     templateObj.definitions.PipelineTaskCondition.properties.conditionRef.enum = conditions;
+//   }
+//   return templateObj;
+// }
```
Comment (evidolob) [inline src/yaml-support/tkn-yaml-scheme-generator.ts:97]: This could be deleted, I think

File: src/yaml-support/tkn-yaml-scheme-generator.ts [hunk 2]
```diff
@@ -190,7 +188,7 @@ async function generate(doc: vscode.TextDocument, schemaPath: string): Promise<s
   const template = await readFile(schemaPath, 'UTF8');
   if (schemaPath.endsWith(path.join('tekton.dev', 'v1beta1_Pipeline.json'))) {
     const snippets = await getTknTasksSnippets();
-    const conditions = await getTknConditionsSnippets();
+    // const conditions = await getTknConditionsSnippets();
```
Comment (evidolob) [inline src/yaml-support/tkn-yaml-scheme-generator.ts:191]: And this

File: src/yaml-support/tkn-yaml-scheme-generator.ts [hunk 3]
```diff
@@ -206,7 +204,7 @@ async function generate(doc: vscode.TextDocument, schemaPath: string): Promise<s
     }
     templateWithSnippets = injectTasksName(templateWithSnippets, definedTasks, tasksRef);
     templateWithSnippets = injectResourceName(templateWithSnippets, resNames);
-    templateWithSnippets = injectConditionRefs(templateWithSnippets, conditions);
+    // templateWithSnippets = injectConditionRefs(templateWithSnippets, conditions);
```
Comment (evidolob) [inline src/yaml-support/tkn-yaml-scheme-generator.ts:207]: And this

**Review by evidolob — APPROVED** [review]

**Pull request merged**